### ISSUE TITLE
Additional constraints for prefix request 

### ISSUE THREAD

**Issue by jamesaoverton** [OBOFoundry/OBOFoundry.github.io#600]
_From @GoogleCodeExporter on August 12, 2015 2:4_

```
The technical group would like to ask the editorial group to formulate a policy 
about (1) resources need to have broad enough scope to request prefix (2) 
shorter prefixes should be allocated to broader scope resources
```

Original issue reported on code.google.com by `mcour...@gmail.com` on 8 Aug 2014 at 9:30

_Copied from original issue: OBOFoundry/Operations-Committee-RETIRED#149_

**Comment by nataled** [comment]
Issues to consider when formulating this policy:
1) What do we consider a 'short' prefix? Two characters? Three?
2) How do we define broad vs narrow scope ontologies?
3) How do we recognize broad vs narrow ontologies?

**Comment by cmungall** [comment]
* Tiny: `[A-Z]O`
 * Short: 3 characters

Broad: applicability across a range of disciplines, or encompasses a whole discipline, where discipline is something you might find as the name of a department in an academic institute.

Narrow: speciality within a discipline. 

It doesn't matter if it's fuzzy. The policy should be that we prefer narrower subjects don't squat on the short names, but this can be negotiated. Thus far everyone has understood and has gracefully complied.

**Comment by nataled** [comment]
@jamesaoverton so far as I can tell this issue was somewhat duplicated and dealt with in the closed issue #954. If you agree, please close this.
